src/offline/cable_mpicommon.F90

```diff
@@ -27,6 +27,13 @@ MODULE cable_mpicommon
 
   PUBLIC
 
+  ! MPI commicator and rank are declared as global variables (see
+  ! cable_driver_init_mod for their initialisation). This is done so that the comm
+  ! and rank variables are only accessible in cable_driver_init_mod when MPI
+  ! compilation is enabled.
+  ! TODO(Sean): revise which module to declare these variables in
+  INTEGER :: comm, rank
```

Comment:
I think you can leave this here for now. Later on we should introduce an object to wrap this kind of MPI information.